core/lib/zksync_core/src/consensus/era.rs

```diff
@@ -0,0 +1,64 @@
+//! This module provides convenience functions to run consensus components in different modes
+//! as expected by the zkSync Era.
+//!
+//! This module simply glues APIs that are already publicly exposed by the `consensus` module,
+//! so in case any custom behavior is needed, these APIs should be used directly.
+
+use std::sync::Arc;
+
+use zksync_concurrency::{ctx, limiter, time};
+use zksync_dal::{ConnectionPool, Core};
+
+use super::{
+    config::{Config, Secrets},
+    fetcher::Fetcher,
+    storage::Store,
+};
+use crate::sync_layer::{sync_action::ActionQueueSender, MainNodeClient, SyncState};
+
+/// Runs the consensus task in the main node mode.
+pub async fn run_main_node(
+    ctx: &ctx::Ctx,
+    cfg: super::MainNodeConfig,
+    pool: ConnectionPool<Core>,
+) -> anyhow::Result<()> {
+    // Consensus is a new component.
+    // For now in case of error we just log it and allow the server
+    // to continue running.
+    if let Err(err) = cfg.run(ctx, Store(pool)).await {
+        tracing::error!(%err, "Consensus actor failed");
+    } else {
+        tracing::info!("Consensus actor stopped");
+    }
+    Ok(())
```

Comment:
I think that this wrapper is still convenient, even if that simple, because we intentionally silence the error here, while maintaining a signature that returns a `Result`. If we'll change this behavior here, we'll change it everywhere. Also, for symmetry with `run_fetcher`.